cw/cw305/mix_columns_cpa_attack.py

```diff
@@ -0,0 +1,33 @@
+#!/usr/bin/env python3
+# Copyright lowRISC contributors.
+# Licensed under the Apache License, Version 2.0, see LICENSE for details.
+# SPDX-License-Identifier: Apache-2.0
+import binascii
+import chipwhisperer as cw
+from chipwhisperer.analyzer.attacks.attack_mix_columns import AttackMixColumns
```

Comment:
In ChipWhisperer v.5.3.1 there is no `attack_mix_columns` in `chipwhisperer.analyzer.attacks`. Can you please confirm the version number @moidx? Maybe we need to update the `python-requirements.txt`.

Reply:
Hi @vogelpi, I used the git installation option on chipwhisperer to be able to get the latest changes from their `develop` branch: https://chipwhisperer.readthedocs.io/en/latest/installing.html#install-repo-git

I created a fork of the repo and used an upstream configuration similar to what we do with OpenTitan repos.

Reply:
Ah I see! Thanks for providing that info. Now the attack is also working in my setup.